src/components/QueryOutput/QueryOutput.js

```diff
@@ -581,7 +477,7 @@ export class QueryOutput extends React.Component {
       if (referenceIdNumber >= 200 && referenceIdNumber < 300) {
         return false
       }
-    } catch (error) {}
+    } catch (error) { }
```

Comment:
may need add something here? or, this catch will actually do nothing.

Reply:
Updated